diff --git a/gyp/pylib/gyp/input.py b/gyp/pylib/gyp/input.py
index 8f39519dee51fb35c2e295a22ae674a1358a37cb..ac0a4a05e604f2bfe86eb7147bc5ee3368383db1 100644
--- a/gyp/pylib/gyp/input.py
+++ b/gyp/pylib/gyp/input.py
@@ -1180,7 +1180,7 @@ def EvalSingleCondition(cond_expr, true_dict, false_dict, phase, variables, buil
         else:
             ast_code = compile(cond_expr_expanded, "<string>", "eval")
             cached_conditions_asts[cond_expr_expanded] = ast_code
-        env = {"__builtins__": {}, "v": Version}
+        env = {"__builtins__": {"openssl_fips": ""}, "v": Version}
         if eval(ast_code, env, variables):
             return true_dict
         return false_dict
